### DISCUSSION TITLE
What determines the size of the requested memory

### DISCUSSION THREAD

**Discussion by hellsge** [nothings/stb#1591] [Q&A]
In function stbtt__hheap_alloc
``` C          
int count = (size < 32 ? 1000 : size < 128 ? 400 : 100);
stbtt__hheap_chunk *c = (stbtt__hheap_chunk *) STBTT_malloc(sizeof(stbtt__hheap_chunk) + size * count, userdata);
```
STBTT_malloc is used to get a piece of memory. Depending on the platform, the size here is different. I know that the size here is the size of the raster structure, and the count here will be adjusted according to this size, I don't understand why the count here is calculated this way. If I reduce the count here, does that have any effect on the raster?

**Comment by MWP** [comment]
I'm confused by this also... does anyone have an explanation for this?
